include/deviceconfig.h

```diff
@@ -326,4 +337,23 @@ class DeviceConfig : public IJSONSerializable
     {
         SetAndSave(rememberCurrentEffect, newRememberCurrentEffect);
     }
+
+    int GetPowerLimit() const
+    {
+        return powerLimit;
+    }
+
+    ValidateResponse ValidatePowerLimit(const String& newPowerLimit)
+    {
+        if (newPowerLimit.toInt() < POWER_LIMIT_MIN)
+            return { false, String("powerLimit is below minimum value of ") + POWER_LIMIT_MIN };
+
+        return { true, "" };
+    }
+
+    void SetPowerLimit(int newPowerLimit)

```

Comment:
Wait, what if the power limit is being set lower?

Reply:
If the power limit is attempted to be set lower than the minimum of 2000, the request will be ignored.